### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by amansethi00**
CSS component Library
netlify link- https://605c4eea9519620007278107--heuristic-heyrovsky-3084b5.netlify.app/
Open for pr review

@Team-Asynchronous @villdev @vaishnavme

### CODE REVIEW

File: css/components/button.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
+@import "../utilities/utilities.css";
+
+button {
```
Comment (vaishnavme) [inline css/components/button.css:3]: Remove outline on buttons

File: css/components/button.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  text-decoration: none;
+}
+.btn-secondary-lg:hover,
+.btn-secondary-md:hover,
+.btn-secondary-sm:hover {
+  background-color: var(--primary-color-hover);
+  color: var(--white);
+}
+
+.btn-floating {
```
Comment (vaishnavme) [inline css/components/button.css:94]: I think floating button is not perfect circle

File: css/components/button.css [hunk 3]
```diff
@@ -0,0 +1,106 @@
+@import "../utilities/utilities.css";
+
+button {
+  cursor: pointer;
```
Comment (vaishnavme) [inline css/components/button.css:4]: - Remove outline on buttons
- Margin between buttons

**Review by vaishnavme — COMMENTED** [review]
Great work Aman. Only thing I think that can be improved is buttons. Everything else works perfect.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,997 @@
...
+        </div>
+
+        
+        
+      </div>
+      <div class="pd-1 mg-bottom-1" id="alert">
+        <h3>Alert</h3>
+        <div class="alert-red mg-bottom-1">
+          <div class="alert-text">Danger alert</div>
+          <img class="alert-image" src="./assets/x-mark.svg">
```
Comment (villdev) [inline index.html]: The close icon should ideally be a button element.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,997 @@
...
+          </code>
+        </pre>
+      </div>
+      </div>
+      <div class="pd-1 mg-bottom-1" id="button">
+        <h3>Button</h3>
+        <div class="row-responsive" >
+          <button class="btn-primary-lg">Primary Button</button>
+          <button class="btn-primary-md mg-top-half mg-bottom-half">Primary Button</button>
+          <button class="btn-primary-sm">Primary Button</button>
```
Comment (villdev) [inline index.html]: Padding seems a little less for `btn-primary-sm`. Maybe increase it or make it similar to other buttons.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,997 @@
...
+        <div class="code">
+          <pre>
+            <code class="language-html">
+              &#x3C;a href=&#x22;&#x22; class=&#x22;btn-primary-md&#x22;&#x3E;Link Button&#x3C;/a&#x3E;
+              &#x3C;a href=&#x22;&#x22; class=&#x22;btn-secondary-md&#x22;&#x3E;Link Button&#x3C;/a&#x3E;
+            </code>
+          </pre>
+        </div>
+        <div class="relative">
+          <button class="btn-floating relative ">+</button>
```
Comment (villdev) [inline index.html]: Give corresponding width as well.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,997 @@
...
+        <br/>
+        <br/>
+      </div>
+      <div class="pd-1 mg-bottom-1" id="card">
+        <h3>Card</h3>
+        <div class="card" style="width:18rem">
+            <img class="card-image" src="https://cdn.discordapp.com/avatars/778699572787675136/bfd4785c0eb8a2be4eae7cc2fec27d34.png?size=256&quot"
+          >
+          <div class="card-body">
+            <div class="card-head">Header</div>
```
Comment (villdev) [inline index.html]: `card-head` seems to need a little bit more emphasis in all cards. Maybe slightly increase the font-weight.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,997 @@
...
+                &#x3C;/div&#x3E;
+              &#x3C;/div&#x3E;
+              </code>
+            </pre>
+          </div>
+
+          <h4>Cards with text overlay</h4>
+          <div class="card" style="width: 15rem;">
+            <img src="https://cdn.discordapp.com/avatars/778699572787675136/bfd4785c0eb8a2be4eae7cc2fec27d34.png?size=256&quot" class="card-image">
+            <div class="card-text-overlay">
```
Comment (villdev) [inline index.html]: Most overlay are usually rgba values of black. This works as well though, just nit picking.

File: node_modules/highlight.js/LICENSE [hunk 9]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```
Comment (villdev) [inline node_modules/highlight.js/LICENSE]: Add `.gitignore` file and exclude `node_modules` directory.

**Review by villdev — COMMENTED** [review]
Good job Aman!

**Comment by Dev-kunal** [comment]
Good Job Aman..!  just a suggestion that colors used for alerts can be improved. You can take inspiration from bootstrap or similar platform.

**Comment by amansethi00** [comment]
> 
> 
> Good Job Aman..! just a suggestion that colors used for alerts can be improved. You can take inspiration from bootstrap or similar platform.

Ok will do that

**Comment by amansethi00** [comment]
> 
> 
> Great work Aman. Only thing I think that can be improved is buttons. Everything else works perfect.

Thank you for the review @vaishnavme ,I worked on your suggestions and recently made a commit for the same.I didn't remove outline from button as I think I will be more comfortable to use it that way(as it will be mostly used by me so I guess it should work fine).

**Comment by amansethi00** [comment]
> 
> 
> Good job Aman!

Thank you @villdev ,I worked on your suggestions and included them all in my recent commit,except the text overlay background color(it looks good so why to change,but next time I will surely remember to use light rgba of black :)

File: css/root.css [hunk 10]
```diff
@@ -0,0 +1,27 @@
+/* @import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;900&display=swap"); */
+
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap");
```
Comment (imrhlrvndrn) [inline css/root.css:3]: Select a set of sizes and discard the rest

File: docs.html [hunk 11]
```diff
@@ -0,0 +1,1008 @@
...
+              <span class="badge-avatar-text-lg">41</span>
+            </div>
+            <div class="card-badge-body">
+              <div class="card-badge-head-lg">Hello</div>
+              <div class="card-badge-text-lg">Some message</div>
+            </div>
+          </div>
+          <div class="card card-badge card-body" style="width: 12rem;">
+            <div class="badge">
+              <img src="./assets//ironman.png" class="avatar-md"/>
```
Comment (imrhlrvndrn) [inline docs.html]: Fix the no.of /

**Review by imrhlrvndrn — CHANGES_REQUESTED** [review]

**Pull request merged**